README.md

```diff
@@ -45,12 +45,25 @@ bash startup-scripts/dev_graphql_dashboard_ui.sh
 # Populate MongoDB with Backup
 When you build the dashboard-mongo container for the first time, it will be empty. Follow these steps to populate it:
 
-1. Navigate to AWS and download backup.tar.gz from s3 bucket (dashboard-assets/mongodb-backups)
+1. Navigate to AWS and download the most recent mongodb backup from S3/Buckets/itm-backups/mongodb
 2. Uncompress the file
 ```
-tar -xzvf backup.tar.gz
+tar -xzvf <file_name>
 ```
-3. Import data into MongoDB
+3. Import data into MongoDB (file should uncompress to dashboard-mongodb.dump)
```

Comment:
Just want to make sure this is new-- that `dashboard-mongodb.dump` is the new standard way to create the backup to store in S3.  Because I don't think it was that way for the backup I used last week (or whenever it was).

Reply:
Yea so I just gave you a copy of my database. This is how to get a fresh copy of production from aws